src/orquestra/sdk/_base/cli/_dorq/_ui/_presenters.py

```diff
@@ -222,6 +224,48 @@ def open_url_in_browser(self, url) -> bool:
         return webbrowser.open(url)
 
 
+class ConfigPresenter:
+    """
+    Present config information to the user.
+    """
+
+    def print_configs_list(
+        self,
+        configs: t.Sequence[RuntimeConfiguration],
+        status: t.Mapping[ConfigName, bool],
+        message: t.Optional[str] = "Stored configs:",
+    ):
+        """
+        Print a list of stored configs.
+        """
+        click.echo(message)
+        click.echo(
+            tabulate(
+                [
+                    [
+                        click.style(config.config_name, bold=True),
+                        click.style(config.runtime_name, fg="blue")
+                        if config.runtime_name == RuntimeName.CE_REMOTE
+                        else click.style(config.runtime_name, fg="green"),
+                        config.runtime_options["uri"],
+                        click.style("\u2713", fg="green")
+                        if status[config.config_name]
+                        else click.style("\u2A09", fg="red"),
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**thought:** ‏This is a bit terse, which may be difficult to read at a glance.

Reply:
I've broken it up with some added comments so it should be a bit more readable now:

```python
            tabulate(
                [
                    [
                        # show config name
                        click.style(config.config_name, bold=True),
                        #
                        # show runtime name, colour coded blue for CE and green for QE
                        click.style(config.runtime_name, fg="blue")
                        if config.runtime_name == RuntimeName.CE_REMOTE
                        else click.style(config.runtime_name, fg="green"),
                        #
                        # show cluster URI
                        config.runtime_options["uri"],
                        #
                        # show a green tick if the token is current, and a red cross if
                        # it is not.
                        click.style("\u2713", fg="green")
                        if status[config.config_name]
                        else click.style("\u2A09", fg="red"),
                    ]
                    for config in configs
                ],
```
([7f24136](https://github.com/zapatacomputing/orquestra-workflow-sdk/pull/206/commits/7f241362a2d4530b7385b30c3064a7c9a8af9a7e))